aws-sdk/aws-sdk.d.ts

```diff
@@ -1072,7 +1072,7 @@ declare module "aws-sdk" {
 			GrantReadACP?: string;
 			GrantWriteACP?: string;
 			Key: string;
-			Metadata?: string[];
+			Metadata?: any;
```

Comment:
I think `{ [key: string]:string;}` is better than `any`;
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#putObject-property
`Metadata — (map<String>) A map of metadata to store with the object in S3.`.